### PR TITLE
Display channels info in charm/bundle detail pages.

### PR DESCRIPTION
**Pull request by frankban**
Fixes https://github.com/juju/juju-gui/issues/2654

### CODE REVIEW

**Comment by frankban** [comment]
QA: visit several charms and bundles in the store, and check that, if the entity is currently published on any channel, the the channels are visible in the header of the page.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6955/

File: jujugui/static/gui/src/app/jujulib/charmstore.js [hunk 1]
```diff
@@ -324,19 +331,20 @@ var module = module;
...
-        'tags',
-        'charm-metrics',
         'owner',
+        'published',
         'resources',
-        'supported-series'
+        'revision-info',
+        'stats',
+        'supported-series',
+        'tags'
```
Comment (Lukewh) [inline jujugui/static/gui/src/app/jujulib/charmstore.js]: Thanks for sorting these

File: jujugui/static/gui/src/app/jujulib/test-charmstore.js [hunk 2]
```diff
@@ -544,14 +554,27 @@ describe('jujulib charmstore', function() {
...
+        '&include=extra-info' +
+        '&include=id-revision' +
+        '&include=manifest' +
+        '&include=owner' +
+        '&include=published' +
+        '&include=resources' +
+        '&include=revision-info' +
+        '&include=stats' +
+        '&include=supported-series' +
+        '&include=tags'
```
Comment (Lukewh) [inline jujugui/static/gui/src/app/jujulib/test-charmstore.js]: +1 for separate lines. Can we maybe change it to a ` string to get rid of the +'s?

File: jujugui/static/gui/src/app/components/entity-details/header/test-header.js [hunk 3]
```diff
@@ -86,6 +87,9 @@ describe('EntityHeader', function() {
                   trusty
                 </li>]}
                 {undefined}
+                {[<li key="stable" className="entity-header__channel">
```
Comment (mitechie) [inline jujugui/static/gui/src/app/components/entity-details/header/test-header.js]: Is this outputting a stable channel as a default value? Is that true?

**Comment by Lukewh** [comment]
I feel like the channels should be within the same li. Eg /juju-gui has:
screen shot 2017-03-20 at 12 01 48
I feel 
screen shot 2017-03-20 at 12 02 34
would be better. To fit more in line with other related parts, such as:
screen shot 2017-03-20 at 12 03 57

**Comment by elvirafigueras** [comment]
+1 with @Lukewh on the grouping. Also if series are capitalised then channel names should be too

**Comment by Lukewh** [comment]
QA OK thanks @frankban

**Comment by frankban** [comment]
Thanks for the reviews!
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**